RecoHGCal/TICL/plugins/SeedingRegionGlobal.cc

```diff
@@ -0,0 +1,28 @@
+// Author: Arabella Martelli, Felice Pantaleo, Marco Rovere - arabella.martelli@cern.ch, felice.pantaleo@cern.ch, marco.rovere@cern.ch
+// Date: 06/2019
+#include <algorithm>
+#include <set>
+#include <vector>
+
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "SeedingRegionGlobal.h"
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "TrackingTools/TrajectoryState/interface/TrajectoryStateTransform.h"
+
+using namespace ticl;
+
+SeedingRegionGlobal::SeedingRegionGlobal(const edm::ParameterSet &conf, edm::ConsumesCollector &sumes)
+    : SeedingRegionAlgoBase(conf, sumes) {}
+
+SeedingRegionGlobal::~SeedingRegionGlobal(){};
+
+void SeedingRegionGlobal::makeRegions(const edm::Event &ev,
+                                      const edm::EventSetup &es,
+                                      std::vector<ticl::TICLSeedingRegion> &result) {
+  // for unseeded iterations create 2 global seeding regions
+  // one for each endcap
+  for (int i = 0; i < 2; ++i) {
+    result.emplace_back(
+        ticl::TICLSeedingRegion({GlobalPoint(0., 0., 0.), GlobalVector(0., 0., 0.), i, -1, edm::ProductID()}));
```

Comment:
see above comment about use of `emplace_back`

Reply:
ok